dist/cms.js

```diff
@@ -77,11 +77,86 @@ function getWordPressScripts() {
   return entries;
 }
 
+/**
+ * Detects the type of WordPress content for the current document.
+ *
+ * @returns {object} Object with fields `contentType`, `postType`, and `taxonomy`.
+ */
+function getWordPressContentType() {
+  const content = {
+    contentType: 'unknown',
```

Comment:
Wouldn't `template` make more sense than `contentType` here?

Reply:
That's worth considering. I would like to not stick too much to the very granular WordPress template hierarchy, as this more about the kind of content used (e.g. "singular" vs "archive" is still at a very high level). But I like the `template` name here, as long as we're aligned this shouldn't directly need to map to a WordPress theme template identifier.

Reply:
Yeah, my concern with `contentType` is it seems more synonymous with `postType`.